app/src/main/res/drawable/nav_urlbar_background.xml

```diff
@@ -8,7 +8,7 @@
         <shape
             android:shape="rectangle">
             <corners android:radius="@dimen/urlbar_corner_radius"/>
-            <solid android:color="@color/photonGrey80" />
+            <solid android:color="#32313c" />
```

Comment:
I got the colours from the mocks but I think our colour palette is going to be changing. Maybe I should hold off on any colour changes until they are all changed together.

Reply:
@dnarcese Which mocks? Do you mind linking me? I'm not sure I've seen them.

Reply:
> I think our colour palette is going to be changing. Maybe I should hold off on any colour changes until they are all changed together.

Good find! That palette change could be a ways off so it wouldn't hurt to ask Bram if this might be a problem sooner!

Reply:
https://github.com/mozilla-mobile/firefox-tv/issues/2468#issue-461305344